### PR TITLE
fix(ui): rectangular black box shadow#4176

### LINKED ISSUE #4176: [🐞 Bug]: Under the footer section the social media icons->follow us on has rectangular box  which makes the icons unimpressive.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Under the footer section, the social media icons->follow us on  has a black rectangular box shadow which make icons unimpressive also the footer section.

### Steps To Reproduce

Step 1: Go to  circuitverse.org.
Step 2: Scroll down to footer section.
Step 3: Under footer section the social media icons of follow us on has a rectangular black box( zoom on the icons little bit to get proper view).

### Expected Behavior

Solution: The icons of follow us on under the footer section should have no box shadow which will give the icons more impressive look as well as footer section.

### Screenshots

Screenshot 2023-10-13 003911

### Device Information [optional]

```markdown
- OS: Windows 11
- Browser: Brave
- version:1.58.137
```

### Additional context

_No response_

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by priyanshuxoxo**
Fixes #4176 

### Describe the changes you have made in this PR -

*Removed the box shadow from the social icons of follow us on under the footer section.

### Screenshots of the changes (If any) -
Screenshot 2023-10-14 143554

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

### CODE REVIEW

**Comment by priyanshuxoxo** [comment]
@vedant-jain03 @tanmoysrt can you please review this PR?

**Review by tanmoysrt — APPROVED** [review]
Looks fine !
cc @vedant-jain03

File: app/assets/stylesheets/footer.scss [hunk 1]
```diff
@@ -32,7 +32,6 @@
 }
 
 .bounce-out-on-hover {
-  box-shadow: 0 0 1px $black;
```
Comment (tanmoysrt) [inline app/assets/stylesheets/footer.scss:35]: Instead of removing it, we can make it transparent.

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/footer.scss [hunk 2]
```diff
@@ -32,7 +32,7 @@
...
```
Comment (vedant-jain03) [inline app/assets/stylesheets/footer.scss:35]: I think if not needed, we can remove the padding that looks a little large between item

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/footer.scss (hunk 2 above)
Comment (tanmoysrt) [reply app/assets/stylesheets/footer.scss:35]: I didn't get it, whether there is any issue with removing the box shadow? Ref https://github.com/CircuitVerse/CircuitVerse/pull/2591

**Review by priyanshuxoxo — COMMENTED** [review]
@tanmoysrt  i check https://github.com/CircuitVerse/CircuitVerse/pull/2591 removing the box shadow doesn't effect the hover effect of icons which were added on purpose ,so removing the box shadow will give the icons more impressive look or rather instead removing we can make it transparent.

**Review by priyanshuxoxo — COMMENTED** [review]
@vedant-jain03  i decreased the padding which was earlier looking little large between items in #4156 .

**Comment by tanmoysrt** [comment]
> @tanmoysrt i check #2591 removing the box shadow doesn't effect the hover effect of icons which were added on purpose ,so removing the box shadow will give the icons more impressive look or rather instead removing we can make it transparent.

That's fine !

**Review by tanmoysrt — APPROVED** [review]

File: app/assets/stylesheets/footer.scss (hunk 2 above)
Comment (vedant-jain03) [reply app/assets/stylesheets/footer.scss:35]: Checked, I find no border much better UX

**Review by vedant-jain03 — APPROVED** [review]

**Comment by vedant-jain03** [comment]
@priyanshuxoxo kindly resolve the conflicts

**Comment by priyanshuxoxo** [comment]
@vedant-jain03  done

**Pull request merged**